nautobot/docs/additional-features/caching.md

```diff
@@ -163,8 +165,9 @@ redis_password = ""
 sentinel_password = ""
 
 CELERY_BROKER_URL = (
-    f"sentinel://:{redis_password}@mysentinel.redis.example.com:26379,",
-    f"sentinel://:{redis_password}@othersentinel.redis.example.com:26379",
+    f"sentinel://:{redis_password}@mysentinel.redis.example.com:26379;"
+    f"sentinel://:{redis_password}@othersentinel.redis.example.com:26379;"
+    f"sentinel://:{redis_password}@thirdsentinel.redis.example.com:26379"
```

Comment:
Should note that the final entry should *not* end with a semicolon. When it is present, Celery tries to append a localhost entry to the list.

Reply:
Per the docs, I did retry using a list and removing the trailing delimiter, but this did not work (as experienced previously).

https://docs.celeryproject.org/en/stable/userguide/configuration.html#broker-url

Reply:
Yeah in some cases (not sure of the specifics) the empty string is interpreted as localhost. This threw me for a loop recently. Not sure on the list, seems like their docs are off...